docs/README.md

```diff
@@ -82,6 +82,16 @@ Your content goes here.
 {% /callout %}
 ```
 
+#### Disclosure
```

Comment:
Minor comment, but can't we just add a "collapsible" option to the existing callouts?

Reply:
Yes, I debated whether I should do it that way.  I'd be happy to reimplement it as the same component, but I did as a separate component because the original callout component markup would have to be rearranged to account for the collapsible bar across the top.

I was erring on the side of repeating code so that different components that happen to look similar can evolve independently.  These two components are very similar and could definitely be implemented as the same component though.